schema/CodeChunk.md

```diff
@@ -87,6 +103,7 @@ declares:
         schema:
           - type: StringSchema
```

Comment:
Think a typo here might've been the culprit for the build issues @100ideas.
The `schema` property expected an `object`, instead of an `array`. The following change should fix the issue:

```suggestion
          type: StringSchema
```